packages/database/src/core/util/libs/parser.ts

```diff
@@ -37,6 +37,25 @@ function decodePath(pathString: string): string {
   return pathStringDecoded;
 }
 
+/**
+ * @param {!string} queryString
+ * @return {!Map<string, string>} key value hash
+ */
+function decodeQuery(queryString: string): Map<string, string> {
```

Comment:
As I just learned in a Firestore PR, IE 11's Map support is kind of lackluster and so it would be better if you used a normal Object here: `{ [ key: string] : string }`